approval_polls/static/approval_polls/embed_instructions.js

```diff
@@ -0,0 +1,26 @@
+/* Code related to templates/approval_polls/embed_instructions.html */
+
+// Sets the iframe height and the corresponding embed instruction code in the textarea.
+function setIframeHeightAndCode(iframeId, textareaId) {
```

Comment:
Should be `setIframeHeightAndCode = function (iframeId, textareaId)` for code consistency and to make hoisting explicit.